core_data_modules/traced_data/util/fold_traced_data.py

```diff
@@ -255,6 +262,27 @@ def reconcile_yes_no_keys(cls, user, td_1, td_2, keys):
         td_1.append_data(yes_no_dict, Metadata(user, Metadata.get_call_location(), time.time()))
         td_2.append_data(yes_no_dict, Metadata(user, Metadata.get_call_location(), time.time()))
 
+    @classmethod
+    def reconcile_binary_keys(cls, user, td_1, td_2, keys):
```

Comment:
I'm afraid I don't find this name self-explanatory - could you add a doc comment please

Reply:
added